### PR TITLE
[Feat] - Migration of Docs from Old Site

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- Migration of Files with respect to https://docs.google.com/document/d/1wYOLvhuWt50M0sMPGN2TlA5JMICzv5BSczPmV4GsilQ/edit

### CODE REVIEW

File: docs/embed/nodejs/_category_.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Node.js SDK for Embedding Wasm Functions",
+  "position": 11,
+  "link": {
+    "type": "generated-index",
+    "description": "In this chapter, we will learn how to Embed WasmEdge in your Node.js applications."
+  }
+}
```
Comment (alabulei1) [inline docs/embed/nodejs/_category_.json:8]: No need to add this one. Because Node.js SDK will be decrypted.

File: docs/embed/nodejs/intro.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
+---
```
Comment (alabulei1) [inline docs/embed/nodejs/intro.md:1]: No need to add this one. Because Node.js SDK will be decrypted.

**Comment by alabulei1** [comment]
I found that you missing WasmEdge runner here. https://wasmedge.org/book/en/cli/wasmedge_tool.html

File: docs/develop/build-and-run/install.md [hunk 3]
```diff
@@ -282,6 +290,28 @@ If you used `winget` to install WasmEdge on Windows, run the following command t
...
+npm install -g wasmedge-extensions # Append --unsafe-perm if permission denied
+```
+
+The [Second State Functions](https://www.secondstate.io/faas/) is a WasmEdge-based FaaS service build on Node.js.
+
+To install certificates in your (MacOS) environment,
+
+1. Find out which Python version you're using with `python --version`
+2. Open the folder /Applications/Python 3.x
+3. Double click the `Install Certificates.command`.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply docs/develop/build-and-run/install.md:313]: @alabulei1 I didn't get you, was it meant to be deprecated? 😶

**Comment by adithyaakrishna** [comment]
@alabulei1 Made the changes and Resolved Conflicts as well :)

File: docs/develop/build-and-run/install.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (alabulei1) [reply docs/develop/build-and-run/install.md:313]: This part should be deleted.

File: docs/develop/build-and-run/run.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+---
+
+# The AoT Compiler
```
Comment (alabulei1) [inline docs/develop/build-and-run/run.md:5]: The title here is not correct.

File: docs/develop/getting-started/quick_start_docker.md [hunk 5]
```diff
@@ -16,6 +16,8 @@ We will cover the following examples.
 - [Run a standalone Wasm app](#run-a-standalone-wasm-app)
 - [Run an HTTP server](#run-an-http-server)
 - [Run a JavaScript server (node.js)](#run-a-javascript-based-server)
+- [Using WasmEdge DockerSlim](#wasmedge-dockerslim)
+- [Docker Images for Building WasmEdge](/docs/contribute/source/docker.md)
```
[the remark this answers is not in the document]
Comment (alabulei1) [reply docs/develop/getting-started/quick_start_docker.md:19]: Delete this line.

File: docs/develop/getting-started/quick_start_docker.md [hunk 6]
```diff
@@ -99,6 +100,64 @@ To learn more about how to run JavaScript apps in WasmEdge.
...
+
+### Examples
+
+Use `wasmedge compile` and `wasmedge` ([link](https://github.com/WasmEdge/WasmEdge/tree/master/examples/wasm)):
+
+```bash
+$ docker pull wasmedge/slim:{{ wasmedge_version }}
+
+$ docker run -it --rm -v $PWD:/app wasmedge/slim:{{ wasmedge_version }} wasmedge compile hello.wasm hello.aot.wasm
+[2022-07-07 08:15:49.154] [info] compile start
```
Comment (alabulei1) [inline docs/develop/getting-started/quick_start_docker.md:132]: Move this part to https://wasmedge.org/docs/category/deploy-wasmedge-apps-in-kubernetes

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
@adithyaakrishna  Please check it out again. Thanks.

**Comment by alabulei1** [comment]
LGTM! Thanks for your contribution.

**Pull request merged**